source/crud/crud.rst

```diff
@@ -884,6 +884,17 @@ Basic
      */
     collation: Optional<Document>;
 
+    /**
+     * The index to use.
```

Comment:
If this is anything like `find` (and I can't imagine why it would differ), it will error if the hint does not exist:

```
rs0:PRIMARY> db.foo.find().hint("_id_")
{ "_id" : ObjectId("5d26307a2a8bc03e915969ad"), "x" : 1 }
rs0:PRIMARY> db.foo.find().hint("_id_dne")
Error: error: {
	"operationTime" : Timestamp(1562783925, 1),
	"ok" : 0,
	"errmsg" : "error processing query: ns=test.fooTree: $and\nSort: {}\nProj: {}\n planner returned error :: caused by :: hint provided does not correspond to an existing index",
	"code" : 2,
	"codeName" : "BadValue",
	"$clusterTime" : {
		"clusterTime" : Timestamp(1562783925, 1),
		"signature" : {
			"hash" : BinData(0,"AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
			"keyId" : NumberLong(0)
		}
	}
}
```

[`find`](https://docs.mongodb.com/manual/reference/command/find/) describes the `hint` option as:

> Optional. Index specification. Specify either the index name as a string or the index key pattern. If specified, then the query system will only consider plans using the hinted index.

Although it's not explicit, "only consider" implies that the query cannot execute if the hinted index does not exist.